tests/operator_tests.py

```diff
@@ -732,10 +732,15 @@ def test_str_repr(self):
         # Test __str__()
         self.assertEqual("[1 2 3]", ak.array([1, 2, 3]).__str__())
         self.assertEqual("[1 2 3 ... 17 18 19]", ak.arange(1, 20).__str__())
-        self.assertEqual("[1.100000e+00 2.300000e+00 5.000000e+00]", ak.array([1.1, 2.3, 5]).__str__())
-        self.assertEqual(
-            "[0.000000e+00 5.263158e-01 1.052632e+00 ... 8.947368e+00 9.473684e+00 1.000000e+01]",
-            ak.linspace(0, 10, 20).__str__(),
+        # Printing floating point values changed precision in Chapel 1.32
+        self.assertTrue(
+            ("[1.100000e+00 2.300000e+00 5.000000e+00]" == ak.array([1.1, 2.3, 5]).__str__()) or
+            ("[1.1 2.3 5.0]" == ak.array([1.1, 2.3, 5]).__str__())
+        )
+        self.assertTrue(
+            ("[0.000000e+00 5.263158e-01 1.052632e+00 ... 8.947368e+00 9.473684e+00 1.000000e+01]" == \
+            ak.linspace(0, 10, 20).__str__()) or
+            ("[0.0 0.526316 1.05263 ... 8.94737 9.47368 10.0]" == ak.linspace(0, 10, 20).__str__())
```

Comment:
this is totally fine, but I think it's a little hard to read. I think at some point i think it would be nice to update it to be something like
```python
# Printing floating point values changed precision in Chapel 1.32
answers = ["[1.100000e+00 2.300000e+00 5.000000e+00]", "[1.1 2.3 5.0]"]
self.assertTrue(ak.array([1.1, 2.3, 5]).__str__() in answers)

answers = [
    "[0.000000e+00 5.263158e-01 1.052632e+00 ... 8.947368e+00 9.473684e+00 1.000000e+01]",
    "[0.0 0.526316 1.05263 ... 8.94737 9.47368 10.0]",
]
self.assertTrue(ak.linspace(0, 10, 20).__str__() in answers)
```

Reply:
You're a genius Pierce...

Reply:
😁